src/components/InstrumentList.tsx

```diff
@@ -0,0 +1,91 @@
+import Box from "@mui/material/Box";
+import List from "@mui/material/List";
+import ListItem from "@mui/material/ListItem";
+import ListItemButton from "@mui/material/ListItemButton";
+import ListItemText from "@mui/material/ListItemText";
+import Divider from "@mui/material/Divider";
+import { Fragment } from "react";
+import ListSubheader from "@mui/material/ListSubheader";
+import Paper from "@mui/material/Paper";
+
+const instruments = [
+  "ALF",
+  "ARGUS",
+  "CHIPLR",
+  "CHRONUS",
+  "CRISP",
+  "EMU",
+  "ENGINX",
+  "GEM",
+  "HET",
+  "HIFI",
+  "HRPD",
+  "IMAT",
+  "INES",
+  "INTER",
+  "IRIS",
+  "LAD",
+  "LARMOR",
+  "LET",
+  "LOQ",
+  "MAPS",
+  "MARI",
+  "MERLIN",
+  "MUSR",
+  "NILE",
+  "NIMROD",
+  "OFFSPEC",
+  "OSIRIS",
+  "PEARL",
+  "PEARL (HIPR)",
+  "POLARIS",
+  "POLREF",
+  "PRISMA",
+  "ROTAX",
+  "SANDALS",
+  "SANS2D",
+  "SURF",
+  "SXD",
+  "TFXA",
+  "TOSCA",
+  "VESUVIO",
+  "WISH",
+  "ZOOM",
+];
+
+export default function InstrumentList() {
+  return (
+    <Box sx={{ width: "100%", maxWidth: 360 }}>
+      <nav aria-label="instrument list">
```

Comment:
I don't think `nav` is appropriate here.  The list items will only control state of the same page when the rest of the functionality is added, they wont be navigation links, which is what the semantic definition if meant to imply if I'm not mistaken?